packages/common/src/lib/common/tools.ts

```diff
@@ -23,6 +23,22 @@ import { maybeCallbackWithError } from './maybeCallback';
 const extend = require('node.extend');
 import { setDefaultResultOrder } from 'dns';
 
+type DockerInformation =
+    | {
+          /** If it is a Docker installation */
+          isDocker: boolean;
+          /** If it is the official Docker image */
+          isOfficial: true;
```

Comment:
Maybe enhance name?just official is not that meaningful in my eyes isOfficialIoBrokerImage